README.md

```diff
@@ -73,13 +75,58 @@ Usage:
   sensu-pagerduty-handler [flags]
 
 Flags:
-  -h, --help           help for sensu-pagerduty-handler
-  -t, --token string   The PagerDuty V2 API authentication token, use default from PAGERDUTY_TOKEN env var
-
+  -d, --dedup-key string            The Sensu event label specifying the PagerDuty V2 API deduplication key, use default from PAGERDUTY_DEDUP_KEY env var
+  -k, --dedup-key-template string   The PagerDuty V2 API deduplication key template, use default from PAGERDUTY_DEDUP_KEY_TEMPLATE env var
```

Comment:
I don't see a `PAGERDUTY_DEDUP_KEY_TEMPLATE` env var in the example handler definition, might that be useful?

Reply:
Updated